The debug agent in exec-maven prevents from starting multiple tasks

The dev friendly profile we added significantly reduced the time for debugging; however, I found one small issue when starting multiple services on the same machine with exec:exec. Only one of the server can be started even though they are not binding to the same port. 

```
ERROR: transport error 202: bind failed: Address already in use
ERROR: JDWP Transport dt_socket failed to initialize, TRANSPORT_INIT(510)
JDWP exit error AGENT_ERROR_TRANSPORT_INIT(197): No transports initialized [:735]
[ERROR] Command execution failed.
```
With investigation, I found that we are using a debug agent in the exec-maven task. I am wondering if we can remove the JDWP as I don't think anybody is doing remote debugging because the light-4j application is just a plain POJO. @243826 Could you please provide your comment on it? 

```
        <profile>
            <id>debug</id>
            <activation>
                <activeByDefault>true</activeByDefault>
            </activation>
            <properties>
                <java.debug.argument>-Xdebug</java.debug.argument>
                <java.debug.agent>-Xrunjdwp:transport=dt_socket,address=8000,server=y,suspend=n</java.debug.agent>
            </properties>
            <build>
                <plugins>
                    <plugin>
                        <groupId>org.codehaus.mojo</groupId>
                        <artifactId>exec-maven-plugin</artifactId>
                        <version>${version.exec-maven}</version>
                        <configuration>
                            <workingDirectory>${project.build.directory}</workingDirectory>
                            <executable>java</executable>
                            <arguments>
                                <argument>-cp</argument>
                                <classpath />
                                <argument>${java.debug.argument}</argument>
                                <argument>${java.debug.agent}</argument>
                                <argument>${project.main.class}</argument>
                            </arguments>
                        </configuration>
                    </plugin>
                </plugins>
            </build>
        </profile>

```

--- a/light-eventuate-4j/src/main/resources/templates/eventuate/hybrid/server/pom.xml.rocker.raw
+++ b/light-eventuate-4j/src/main/resources/templates/eventuate/hybrid/server/pom.xml.rocker.raw
@@ -394,10 +394,6 @@
             <activation>
                 <activeByDefault>true</activeByDefault>
             </activation>
-            <properties>
-                <java.debug.argument>-Xdebug</java.debug.argument>
-                <java.debug.agent>-Xrunjdwp:transport=dt_socket,address=8000,server=y,suspend=n</java.debug.agent>
-            </properties>
             <build>
                 <plugins>
                     <plugin>
@@ -410,8 +406,6 @@
                             <arguments>
                                 <argument>-cp</argument>
                                 <classpath />
-                                <argument>${java.debug.argument}</argument>
-                                <argument>${java.debug.agent}</argument>
                                 <argument>${project.main.class}</argument>
                             </arguments>
                         </configuration>

--- a/light-eventuate-4j/src/main/resources/templates/eventuate/hybrid/service/pom.xml.rocker.raw
+++ b/light-eventuate-4j/src/main/resources/templates/eventuate/hybrid/service/pom.xml.rocker.raw
@@ -384,10 +384,6 @@
             <activation>
                 <activeByDefault>true</activeByDefault>
             </activation>
-            <properties>
-                <java.debug.argument>-Xdebug</java.debug.argument>
-                <java.debug.agent>-Xrunjdwp:transport=dt_socket,address=8000,server=y,suspend=n</java.debug.agent>
-            </properties>
             <build>
                 <plugins>
                     <plugin>
@@ -400,8 +396,6 @@
                             <arguments>
                                 <argument>-cp</argument>
                                 <classpath />
-                                <argument>${java.debug.argument}</argument>
-                                <argument>${java.debug.agent}</argument>
                                 <argument>${project.main.class}</argument>
                             </arguments>
                         </configuration>

--- a/light-eventuate-4j/src/main/resources/templates/eventuate/rest/openapi/pom.xml.rocker.raw
+++ b/light-eventuate-4j/src/main/resources/templates/eventuate/rest/openapi/pom.xml.rocker.raw
@@ -546,10 +546,6 @@
             <activation>
                 <activeByDefault>true</activeByDefault>
             </activation>
-            <properties>
-                <java.debug.argument>-Xdebug</java.debug.argument>
-                <java.debug.agent>-Xrunjdwp:transport=dt_socket,address=8000,server=y,suspend=n</java.debug.agent>
-            </properties>
             <build>
                 <plugins>
                     <plugin>
@@ -562,8 +558,6 @@
                             <arguments>
                                 <argument>-cp</argument>
                                 <classpath />
-                                <argument>${java.debug.argument}</argument>
-                                <argument>${java.debug.agent}</argument>
                                 <argument>${project.main.class}</argument>
                             </arguments>
                         </configuration>

--- a/light-graphql-4j/src/main/resources/templates/graphql/pom.xml.rocker.raw
+++ b/light-graphql-4j/src/main/resources/templates/graphql/pom.xml.rocker.raw
@@ -452,10 +452,6 @@
             <activation>
                 <activeByDefault>true</activeByDefault>
             </activation>
-            <properties>
-                <java.debug.argument>-Xdebug</java.debug.argument>
-                <java.debug.agent>-Xrunjdwp:transport=dt_socket,address=8000,server=y,suspend=n</java.debug.agent>
-            </properties>
             <build>
                 <plugins>
                     <plugin>
@@ -468,8 +464,6 @@
                             <arguments>
                                 <argument>-cp</argument>
                                 <classpath />
-                                <argument>${java.debug.argument}</argument>
-                                <argument>${java.debug.agent}</argument>
                                 <argument>${project.main.class}</argument>
                             </arguments>
                         </configuration>

--- a/light-hybrid-4j/src/main/resources/templates/hybrid/server/pom.xml.rocker.raw
+++ b/light-hybrid-4j/src/main/resources/templates/hybrid/server/pom.xml.rocker.raw
@@ -366,10 +366,6 @@
             <activation>
                 <activeByDefault>true</activeByDefault>
             </activation>
-            <properties>
-                <java.debug.argument>-Xdebug</java.debug.argument>
-                <java.debug.agent>-Xrunjdwp:transport=dt_socket,address=8000,server=y,suspend=n</java.debug.agent>
-            </properties>
             <build>
                 <plugins>
                     <plugin>
@@ -382,8 +378,6 @@
                             <arguments>
                                 <argument>-cp</argument>
                                 <classpath />
-                                <argument>${java.debug.argument}</argument>
-                                <argument>${java.debug.agent}</argument>
                                 <argument>${project.main.class}</argument>
                             </arguments>
                         </configuration>

--- a/light-hybrid-4j/src/main/resources/templates/hybrid/service/pom.xml.rocker.raw
+++ b/light-hybrid-4j/src/main/resources/templates/hybrid/service/pom.xml.rocker.raw
@@ -350,10 +350,6 @@
             <activation>
                 <activeByDefault>true</activeByDefault>
             </activation>
-            <properties>
-                <java.debug.argument>-Xdebug</java.debug.argument>
-                <java.debug.agent>-Xrunjdwp:transport=dt_socket,address=8000,server=y,suspend=n</java.debug.agent>
-            </properties>
             <build>
                 <plugins>
                     <plugin>
@@ -366,8 +362,6 @@
                             <arguments>
                                 <argument>-cp</argument>
                                 <classpath />
-                                <argument>${java.debug.argument}</argument>
-                                <argument>${java.debug.agent}</argument>
                                 <argument>${project.main.class}</argument>
                             </arguments>
                         </configuration>

--- a/light-rest-4j/src/main/resources/templates/rest/openapi/pom.xml.rocker.raw
+++ b/light-rest-4j/src/main/resources/templates/rest/openapi/pom.xml.rocker.raw
@@ -472,10 +472,6 @@
             <activation>
                 <activeByDefault>true</activeByDefault>
             </activation>
-            <properties>
-                <java.debug.argument>-Xdebug</java.debug.argument>
-                <java.debug.agent>-Xrunjdwp:transport=dt_socket,address=8000,server=y,suspend=n</java.debug.agent>
-            </properties>
             <build>
                 <plugins>
                     <plugin>
@@ -488,8 +484,6 @@
                             <arguments>
                                 <argument>-cp</argument>
                                 <classpath />
-                                <argument>${java.debug.argument}</argument>
-                                <argument>${java.debug.agent}</argument>
                                 <argument>${project.main.class}</argument>
                             </arguments>
                         </configuration>

--- a/light-rest-4j/src/main/resources/templates/rest/swagger/pom.xml.rocker.raw
+++ b/light-rest-4j/src/main/resources/templates/rest/swagger/pom.xml.rocker.raw
@@ -460,10 +460,6 @@
             <activation>
                 <activeByDefault>true</activeByDefault>
             </activation>
-            <properties>
-                <java.debug.argument>-Xdebug</java.debug.argument>
-                <java.debug.agent>-Xrunjdwp:transport=dt_socket,address=8000,server=y,suspend=n</java.debug.agent>
-            </properties>
             <build>
                 <plugins>
                     <plugin>
@@ -476,8 +472,6 @@
                             <arguments>
                                 <argument>-cp</argument>
                                 <classpath />
-                                <argument>${java.debug.argument}</argument>
-                                <argument>${java.debug.agent}</argument>
                                 <argument>${project.main.class}</argument>
                             </arguments>
                         </configuration>

